guide/de/ch_dep.xml

```diff
@@ -10,222 +10,209 @@
+        abzüglich Abschreibung) gleich dem Restwert ist. Wenn die Anlage dann abgeschrieben wird, entsteht kein Gewinn oder Verlust
+        aus dem Abgang des Vermögenswerts.
       </para>
 
-      <para>The last step is to determine the method of depreciation that you want to use. This will be
-        discussed on the next few pages.
+      <para>Der letzte Schritt besteht darin, die Abschreibungsmethode zu bestimmen, die Sie verwenden möchten. Dies wird
+        auf den nächsten Seiten besprochen.
       </para>
 
       <warning>
-        <para>Be aware that different countries can have substantially different tax policies for depreciation;
-          all that this document can really provide is some of the underlying ideas to help you
-          apply your <quote>favorite</quote> tax/depreciation policies.
+        <para>Seien Sie sich bewusst, dass verschiedene Länder eine sehr unterschiedliche Steuerpolitik für Abschreibungen haben
+          können. Alles, was dieses Dokument wirklich bieten kann, sind einige der zugrunde liegenden Ideen, die Ihnen helfen,
+          Ihre <quote>bevorzugte</quote> Steuer-/Abschreibungspolitik anzuwenden.
         </para>
       </warning>
     </sect2>
   </sect1>
 
   <sect1 id="dep_value1">
-    <title>Estimating Valuation</title>
+    <title>Schätzung der Bewertung</title>
 
-    <para>A central issue with depreciation is to determine how you will estimate the future value of the
-      asset. Compared to the often uncertain estimates one has to do where appreciation of assets is
-      concerned, we are on somewhat firmer ground here. Using sources listed below should make it
-      fairly straight forward to estimate the future value of your depreciating assets.
+    <para>Ein zentrales Thema bei der Abschreibung ist die Frage, wie Sie den zukünftigen Wert des Vermögenswertes schätzen.
+      Im Vergleich zu den oft unsicheren Schätzungen, die man bei der Wertsteigerung von Vermögenswerten machen muss, befinden wir uns hier
+      auf etwas festerem Boden. Die Verwendung der unten aufgeführten Quellen sollte es ermöglichen, die Schätzung des zukünftigen
+      Wertes Ihrer abzuschreibenden Anlagen recht einfach machen.
     </para>
 
     <itemizedlist>
       <listitem>
-        <para><emphasis>Tax Codes:</emphasis> For businesses that want to use depreciation for tax purposes,
-          governments tend to set up precise rules as to how you are required to calculate
-          depreciation. Consult your local tax codes, which should explicitly state how to estimate
-          depreciation.
+        <para><emphasis>Abgabenordnung:</emphasis> Für Unternehmen, die Abschreibungen für steuerliche Zwecke nutzen wollen,
+          neigen Regierungen dazu, genaue Regeln aufzustellen, wie Sie die Abschreibung zu berechnen haben. Konsultieren Sie Ihre
+          lokalen Steuergesetze, die explizit angeben sollten, wie Sie die Abschreibung vorzunehmen haben.
```

Comment:
Sollten wir vielleicht https://www.bundesfinanzministerium.de/Web/DE/Themen/Steuern/Steuerverwaltungu-Steuerrecht/Betriebspruefung/AfA_Tabellen/afa_tabellen.html verlinken?

Reply:
@fellen: Ich muss beruflich öfter mal auf Referenzseiten der BaFin und der Dt. Rentenversicherung verlinken. Das ist übel, weil die die Seiten häufig umbauen und Links dann ungültig sind. Da wir Gnucash dokumentieren, reicht es m.E., dass der Link in Wikipedia bei "AfA" enthalten ist.

Reply:
Ich denke, https://de.wikipedia.org/wiki/Absetzung_f%C3%BCr_Abnutzung ist für ESt  in DE OK, da gibt es einen entsprechenden Link.  Also
&url-wp-de;Absetzung_f%C3%BCr_Abnutzung

Allgemener ist https://de.wikipedia.org/wiki/Abschreibung
Da wird dann auch für einen Backport auch eine Verweis auf IAS 16.62 bzw. IAS 38.98 und US-GAAP